src/Core/Components/DataGrid/FluentDataGridCell.razor.cs

```diff
@@ -24,12 +24,18 @@ public partial class FluentDataGridCell<TGridItem> : FluentComponentBase
     [Parameter]
     public DataGridCellType? CellType { get; set; } = DataGridCellType.Default;
 
+    /// <summary>
+    /// Gets or sets the column of the cell.
+    /// </summary>
+    [Parameter]
+    public ColumnBase<TGridItem>? Column { get; set; }
```

Comment:
I'm not sure that adding this property is a good idea.

I'm not sure adding this property is a good idea. Placing an additional Column reference for all the cells in the **DataGrid**, which can contain  several tens of thousands of cells, could lead to performance problems.

I did not verify but you could use the column index to retrieve the Column: updating the `FluentDataGridRow.Owner` property to `internal` and you will have an access to the Grid and the list of columns: `Owner.Owner.Grid...`.

Reply:
Was a bit more complicated than that but I followed this idea :D